pkg/providers/tinkerbell/create.go

```diff
@@ -146,6 +146,7 @@ func (p *Provider) SetupAndValidateCreateCluster(ctx context.Context, clusterSpe
 		clusterSpecValidator.Register(NewIPNotInUseAssertion(p.netClient))
 	}
 
+	clusterSpecValidator.Register(AssertTinkerbellIPNotInUse(p.netClient))
```

Comment:
i will also move this under this `	if !p.skipIpCheck {` condition
```suggestion
		clusterSpecValidator.Register(NewIPNotInUseAssertion(p.netClient))
		clusterSpecValidator.Register(AssertTinkerbellIPNotInUse(p.netClient))
	}
```